core/driver/sources/src/test/resources/reference.conf

```diff
@@ -0,0 +1,81 @@
+# Default configuration for the org.virtuslab.ideprobe.config.IdeProbeConfig
```

Comment:
Actually... isn't `core/driver/sources/src/main/resources/reference.conf` already on **test** classpath? does this file need to be copied to src/test/resources? (as I'm assuming that's the same file contents?)

Reply:
Yes, the same contents. Hm, I am not sure, but I can give it a try (unfortunatelly cannot do tests locally, so let me push a commit later to test it)

Reply:
Or alternatively if `test` classpath doesn't cover `main` resources, use symlinks, they will probably work as expected

Reply:
> unfortunatelly cannot do tests locally, so let me push a commit later to test it

You can just check with `Source.fromResource` in a dummy test method whether the given resource is visible on the classpath

Reply:
Ok, I removed this reference.conf from test resources and in fact it works